### DISCUSSION TITLE
Radiation and Convection Losses

### DISCUSSION THREAD

**Discussion by nourriahi** [idaholab/moose#24729] [Q&A General] [answered]
Hello everyone,

I am trying to model heat conduction using a laser source at the top, as far this is working, I wanted to include convection and radiation losses on the top to account for laser energy losses. I was wondering if moose has developed something to take this into account since I did not find anything so far?

Thanks
Nour

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can look at adding terms to the boundary condition on top. See 
https://mooseframework.inl.gov/source/bcs/ADRadiativeHeatFluxBC.html
https://mooseframework.inl.gov/source/bcs/ADConvectiveHeatFluxBC.html

Guillaume
